django_etebase/exceptions.py

```diff
@@ -12,37 +10,42 @@ def __init__(self, code, detail, status_code=status.HTTP_400_BAD_REQUEST):
         self.status_code = status_code
 
 
-class AuthenticationFailed(HTTPException):
+class CustomHttpException(Exception):
+    def __init__(self, detail: str, status_code: int, code: str):
```

Comment:
Could you use the same parameter order as what's already there (`EtebaseValidation`) and have the status_code default to 400?

Reply:
sure that makes sense